health/egov-persister/attendance-service-persister.yml

```
@@ -0,0 +1,410 @@
+   name: attendance
+   description: Persists attendance details in eg_wms_attendance_register table
+   fromTopic: save-attendance-health
+   isTransaction: true
+   isAuditEnabled: true
+   module: ATND
+   objecIdJsonPath: $.id
+   tenantIdJsonPath: $.tenantId
+   transactionCodeJsonPath: $.registerNumber
+   auditAttributeBasePath: $.attendanceRegister.*
+   queryMaps:
+    - query: INSERT INTO health.eg_wms_attendance_register(id,tenantid,registernumber,name,startdate,enddate,status,additionaldetails,createdby,lastmodifiedby,createdtime,lastmodifiedtime,referenceid,servicecode,localitycode,reviewstatus) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?);
```

yamllint (1.35.1)

[warning] wrong indentation: expected 3 but found 4

(indentation)